filters.txt

```diff
@@ -702,6 +702,7 @@ ulozisko.sk#@##reklama_big
 ##.reklama_title_flat
 ##.reklamni-up-1
 ##.reklamni-up-2
+##.rklm
```

Comment:
Nejaka ukazka webu, na kterych se neco takoveho nachazi? Jedna se o nejaky vetsi system, ze je to bez urceni domeny?

Reply:
videl som to uz na tucte webov, ako priklad mozem uviest poradne ( poradna.net ) ... podobne je to na pctuning aj na par dalsich weboch.